indy_node/test/write_permission/test_revocation_write_permission.py

```diff
@@ -5,57 +5,105 @@
                                       claim_def_req['operation'][CLAIM_DEF_SIGNATURE_TYPE],
                                       claim_def_req['operation'][CLAIM_DEF_TAG]
                                       ).decode()
-    revoc_req = sdk_sign_request_from_dict(looper, sdk_wallet_client, revoc_reg['operation'])
+    revoc_req = sdk_sign_request_from_dict(looper, wallet, revoc_reg['operation'])
     _, revoc_reply = sdk_send_and_check([json.dumps(revoc_req)], looper, txnPoolNodeSet, sdk_pool_handle)[0]
     return revoc_req
 
 
-def test_client_can_send_revoc_reg_def(client_send_revoc_reg_def):
-    pass
+def test_client_cant_send_revoc_reg_def(looper,
+                                        txnPoolNodeSet,
+                                        sdk_wallet_client,
+                                        sdk_pool_handle,
+                                        build_revoc_def_by_default,
+                                        claim_def, tconf):
+    with pytest.raises(RequestRejectedException):
+        send_revoc_reg_def(looper, txnPoolNodeSet, sdk_pool_handle, build_revoc_def_by_default,
+                           claim_def, sdk_wallet_client)
 
 
-def test_client_can_send_revoc_reg_entry(looper,
-                                         client_send_revoc_reg_def,
-                                         sdk_wallet_client,
-                                         txnPoolNodeSet,
-                                         sdk_pool_handle):
-    revoc_def_req = client_send_revoc_reg_def
-    rev_reg_entry = build_revoc_reg_entry_for_given_revoc_reg_def(revoc_def_req)
-    rev_reg_entry[VALUE][REVOKED] = [1, 2, 3, 4, 5]
-    del rev_reg_entry[VALUE][PREV_ACCUM]
-    rev_entry_req = sdk_sign_request_from_dict(looper, sdk_wallet_client, rev_reg_entry)
-    sdk_send_and_check([json.dumps(rev_entry_req)], looper, txnPoolNodeSet, sdk_pool_handle)
\ No newline at end of file
+def test_allowed_roles_can_send_revoc_reg_def(looper,
+                                              txnPoolNodeSet,
+                                              sdk_wallet_trustee,
+                                              sdk_wallet_trust_anchor,
+                                              sdk_wallet_steward,
+                                              sdk_pool_handle,
+                                              build_revoc_def_by_default,
+                                              claim_def, tconf):
+    # trust anchor
+    send_revoc_reg_def(looper, txnPoolNodeSet, sdk_pool_handle, build_revoc_def_by_default,
+                       claim_def, sdk_wallet_trust_anchor)
+    # steward
+    send_revoc_reg_def(looper, txnPoolNodeSet, sdk_pool_handle, build_revoc_def_by_default,
+                       claim_def, sdk_wallet_steward)
+    # trustee
+    send_revoc_reg_def(looper, txnPoolNodeSet, sdk_pool_handle, build_revoc_def_by_default,
+                       claim_def, sdk_wallet_trustee)
+
+
```

Comment:
Please also add the following tests:
1) `test_not_owner_cant_create_revoc_reg_entry`. In this test try to create RevocRegEntry by 
- a client 
- a trust anchor who is not the author of the corresponding RevocRegDef
- a steward who is not the author of the corresponding RevocRegDef
- a trustee who is not the author of the corresponding RevocRegDef

2) `test_owner_can_edit_revoc_reg_entry`
In this test create a RevocRegEntry, and edit it by the same author.

3) `test_not_owner_cant_edit_revoc_reg_entry`
In this test create a RevocRegEntry by a Trust Anchor
Try to edit it by 
- a client 
- another trust anchor who is not the author of this RevocRegEntry
- a steward who is not the author of this RevocRegEntry
- a trustee who is not the author of this RevocRegEntry